tests/conftest.py

```diff
@@ -111,37 +118,37 @@ def firefox_options(
 ) -> typing.Any:
     """Setup Firefox"""
     firefox_options.log.level = "trace"
-    if request.config.getoption("--run-update-test"):
-        # if request.node.get_closest_marker(
-        #     "update_test"
-        # ):  # disable test needs different firefox
-        #     binary = Path(
-        #         "utilities/firefox-old-nightly-disable-test/firefox/firefox-bin"
-        #     ).absolute()
-        #     firefox_options.binary = f"{binary}"
-        #     firefox_options.add_argument("-profile")
-        #     firefox_options.add_argument(
-        #         f'{Path("utilities/klaatu-profile-disable-test").absolute()}'
-        #     )
-        # else:
-        binary = Path(
-            "utilities/firefox-old-nightly/firefox/firefox-bin"
-        ).absolute()
-        firefox_options.binary = f"{binary}"
-        firefox_options.add_argument("-profile")
-        firefox_options.add_argument(setup_profile)
-    if request.config.getoption("--run-firefox-release"):
-        binary = Path("utilities/firefox-release/firefox/firefox-bin").absolute()
-        firefox_options.binary = f"{binary}"
-    if request.node.get_closest_marker("reuse_profile") and not request.config.getoption(
-        "--run-update-test"
-    ):
-        firefox_options.add_argument("-profile")
-        firefox_options.add_argument(setup_profile)
+    # if request.config.getoption("--run-update-test"):
```

Comment:
Is this the aforementioned complex code in need of simplifying?

Reply:
Correct